src/libraries/System.Drawing.Common/ref/System.Drawing.Common.csproj

```diff
@@ -5,15 +5,12 @@
     <IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>
     <Nullable>enable</Nullable>
   </PropertyGroup>
-  <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
-  <PropertyGroup>
-    <IsPartialFacadeAssembly Condition="'$(TargetFramework)' == 'net461'">true</IsPartialFacadeAssembly>
-  </PropertyGroup>
   <ItemGroup>
-    <Compile Include="System.Drawing.Common.cs" />
+    <Compile Include="System.Drawing.Common.cs" Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework'" />
```

Comment:
Everywhere else you are conditioning on TargetFramework directly. For consistency you want to do this here (and in L12&L13) as well as there is only one .NET Framework and one .NET Standard tfm. In cases where there are multiple, this would of course make sense.

Reply:
I was trying to follow as consistent of a pattern as I could across the change but also wanted to honor the style of the project I was modifying if there is a clear pattern established.

Did you have a suggestion here?  Were you saying that I should prefer a `TargetFramework` comparison when we only needed to apply the condition to a single value?

Reply:
Right. Usually we condition on `TargetFramwork` directly if there is only a single tfm per universe and only use the inferred properties (TFI, TFV) when conditioning on multiple tfms of a universe. In this case this is a big NIT. Feel free to just ignore my feedback :)

Reply:
I've actually discussed with folks before (I think it was @zsd4yr) about style-cop on projects/targets.  It'd be nice if we could a) define a set of unambiguous style guidelines b) codify enforcement of those.